test/common-utils.spec.coffee

```diff
@@ -74,16 +75,12 @@ describe 'Common Utils unit tests', ->
 
     it 'should split a list of skus that is to big for a single query
     into chunks that are small enough for a query', ->
-      # 3 bytes string
-      sku = 'SKU'
       skus = []
-      for i in [1..10000]
-        skus.push(sku)
+      for i in [1..100]
```

Comment:
well since I am generating a string with the length of `i` the `randomString.generate` function took quite some time to generate a random string with up to 10000 chars (obviously). The splitting itself is not slow.